src/ZPublisher/HTTPRequest.py

```diff
@@ -46,12 +46,10 @@
 
 if PY3:
     from html import escape
-    from urllib.parse import splitport
-    from urllib.parse import splittype
+    from urllib.parse import urlparse
 else:
     from cgi import escape
-    from urllib import splitport
-    from urllib import splittype
+    from urlparse import urlparse
```

Comment:
You could also use `from six.moves.urllib.parse import urlparse` to get around the try/except, which `isort` ignores for import sorting.